reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
```

Comment:
RFC1035 also refers to parts of the resource record format as a "Field". Maybe this should be MessageHeader or more specific than FieldName.